PYTHON/DATA/DataPreparation.py

```diff
@@ -0,0 +1,59 @@
+'''
+Installing libraries (if needed)
+'''
+pip install keras
+pip install tensorflow
+pip install theano
+pip install sklearn
+
+'''
+Import 
+'''
+import pandas as pd
+import numpy as np
+from numpy import array
+
+'''
+Before a univariate series can be modeled, it must be prepared.
+
+The LSTM model will learn a function that maps a sequence of past observations as input to an output observation. As such, the sequence of observations must be transformed into multiple examples from which the LSTM can learn.
+
+Consider a given univariate sequence:
+
+[10, 20, 30, 40, 50, 60, 70, 80, 90]
+
+We can divide the sequence into multiple input/output patterns called samples, where three time steps are used as input and one time step is used as output for the one-step prediction that is being learned.
+
+X				y
```

Comment:
This line does not look aligned with the code below...

Reply:
Ok! I will fix the alignment! Thank you! 

Regarding your second comment, I have to say that there are some standard rules of developing ML or DL models in Data Science, including the naming of variables, functions, etc. 

To understand better the idea of naming variables, one should better understand how Ml or DL models work. Very roughly speaking, X is a table in which each column corresponds to some feature of the data; y is a value of feature we want to predict (can be housing price, salaries of workers, a profit of the company, etc.) 

The standard initialization of the data is X - for the initial dataset (treating as a matrix), y - for the initial solution (treating as a vector).  

The model will study data and solution and when new data comes, the model should give the solution based on the data it has been studied on. Since we have the only X as all available data, to test the performance of the model, we should keep some part of the dataset apart from the model and give to the model only some part of the existing data for the exploration. It is a common practice to  split dataset X into X_train and X_test (named exactly like this) and y into y_train and y_test (depending on the size of your dataset in 0.1 or 0.2 proportion) and also comparing the performance of the model getting the output with the name y_pred with the initial solution which were not used for training the model (y_test).

Also, the common practice is to keep the name of the model in accordance with the type of problem you are dealing with - either regressor or classificator. 

I hope now it is clear!

Reply:
Wow, thank you for the detailed explanation! Good to know! 😄